### ISSUE TITLE
Extension cannot work?

### ISSUE THREAD

**Issue by 456ken** [456ken/Highlighter-vscode#11]
I think this extension worked until yesterday.
Today, the all commands in this extension are not able to work displayed 'command *** not found'.

**Comment by 456ken** [comment]
I understanded.
V1.0.1 is only able to work when 'multicolorhighlighter.savelist' option is existed.
The following is the reason for this problem.
``` typescript
	load(): boolean {
		var config = vscode.workspace.getConfiguration("multicolorhighlighter");
		if (config === undefined) {
			return false;
		}

		var savelist: SaveList[] | undefined = config.get("savelist");
		if (savelist === undefined || savelist.length < 1) {
			return false;
		}

==>		savelist.forEach(obj => {
			let highlighter = new Highlighter(this._colorset.filter(value => value.name === obj.color)[0], []);
			obj.keyword.forEach(key => highlighter.add(key));
			this.data.push(highlighter);
		});
```

I will fix this problem in 24 hours.

**Comment by 456ken** [comment]
``` typescript
	load(): boolean {
		const mchconfig = vscode.workspace.getConfiguration("multicolorhighlighter");
		if (mchconfig === undefined) {
			return false;
		}

		if (mchconfig.has("savelist") === false) {
			return false;
		}

		const savelist = <SaveList[]>mchconfig.get("savelist");
		const implementsSaveList = function (params: any): params is SaveList[] {
			return (params !== null &&
				typeof params === "object" &&
				isArray(params) &&
				typeof params[0].color === "string" &&
				typeof params[0].keyword === "object");
		}
		if (!implementsSaveList(savelist)) {
			return false;
		}

		savelist.forEach(obj => {
			let highlighter = new Highlighter(this._colorset.filter(value => value.name === obj.color)[0], []);
			obj.keyword.forEach(key => highlighter.add(key));
			this.data.push(highlighter);
		});
```

**Comment by 456ken** [comment]
Fixed at v1.0.2 released 9-Oct-2019.
